backend/controllers/email.controller.js

```diff
@@ -2,54 +2,67 @@ const nodemailer = require("nodemailer");
 require("dotenv").config();
 
 class EmailController {
+  constructor() {
+    // Set up reCAPTCHA with your site key and secret key
+    this.recaptcha = new RecaptchaV2(process.env.RECAPTCHA_SITE_KEY, process.env.RECAPTCHA_SECRET_KEY);
```

Comment:
how exactly to you setup the env variable for this? can you add a reference to the README for this?

Reply:
Okk I have made the changes to the .env file in the README and added the recaptcha fields, do you also need me to write some sort of explanation or anything in the README for it?

Reply:
Hi yes please, if you can mayb add a bit more detail on where the key is generated and so on.